### ISSUE TITLE
Update Eventbrite documentation to include instructions for multiple registration workshop

### ISSUE THREAD

**Issue by MikeTrizna** [SmithsonianWorkshops/si-carpentries-handbook#5]
The current docs tell how to add in the *singular* Eventbrite ID to build the registration embed, but need to provide guidance on manually adding multiple iFrames.

**Comment by jls943** [comment]
Yes, this would be helpful. An example is in the workshop from December 2021, but I must not have copied it correctly the first time I tried it.
